modules/transactions.js

```diff
@@ -645,10 +654,6 @@ Transactions.prototype.shared = {
 		return __private.getPooledTransactions('getMultisignatureTransactionList', req, cb);
 	},
 
-	getUnconfirmedTransaction: function (req, cb) {
-		return __private.getPooledTransaction('getUnconfirmedTransaction', req, cb);
-	},
-
 	getUnconfirmedTransactions: function (req, cb) {
```

Comment:
can we still get a single transaction?

Reply:
if you call `getUnconfirmedTransactions(1)` you will get one transaction from pool